examples/inverse/plot_mixed_source_space_inverse.py

```diff
@@ -52,22 +49,19 @@
               'Right-Thalamus-Proper',
               'Right-Cerebellum-Cortex']
 
-# Get a surface-based source space. We could set one up like this::
-#
-#     >>> src = setup_source_space(subject, fname=None, spacing='oct6',
-#                                  add_dist=False, subjects_dir=subjects_dir)
-#
-# But we already have one saved:
-
-src = mne.read_source_spaces(op.join(bem_dir, 'sample-oct-6-src.fif'))
+# Get a surface-based source space, here with few source points for speed:
+src = mne.setup_source_space(subject, spacing='oct5',
+                             add_dist=False, subjects_dir=subjects_dir)
 
 # Now we create a mixed src space by adding the volume regions specified in the
 # list labels_vol. First, read the aseg file and the source space bounds
 # using the inner skull surface (here using 10mm spacing to save time):
```

Comment:
sorry to be a bit later here but I am afraid people with just copy paste these defaults which are here due to running time. Can you make more explicit that `oct5` and `pos=10` are not recommended for real analysis? thx